docs/_sass/_font-awesome.scss

```diff
@@ -1,11 +1,10 @@
-/*!
- *  Font Awesome 4.4.0 by @davegandy - http://fontawesome.io - @fontawesome
- *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
- */
 @font-face {
   font-family: 'FontAwesome';
-  src: url('../fonts/fontawesome-webfont.eot?v=4.4.0');
-  src: url('../fonts/fontawesome-webfont.eot?#iefix&v=4.4.0') format('embedded-opentype'), url('../fonts/fontawesome-webfont.woff2?v=4.4.0') format('woff2'), url('../fonts/fontawesome-webfont.woff?v=4.4.0') format('woff'), url('../fonts/fontawesome-webfont.ttf?v=4.4.0') format('truetype'), url('../fonts/fontawesome-webfont.svg?v=4.4.0#fontawesomeregular') format('svg');
+  src: url('../fonts/FontAwesome.eot?9h6hxj');
+  src: url('../fonts/FontAwesome.eot?9h6hxj#iefix') format('embedded-opentype'),
+    url('../fonts/FontAwesome.ttf?9h6hxj') format('truetype'),
+    url('../fonts/FontAwesome.woff?9h6hxj') format('woff'),
+    url('../fonts/FontAwesome.svg?9h6hxj#FontAwesome') format('svg');
```

Comment:
WOFF [is supported by 92% of browsers](http://caniuse.com/#feat=woff). What users are you targeting with all these new files?